asf_core_data/pipeline/preprocessing/feature_engineering.py

```diff
@@ -234,23 +234,19 @@ def get_heating_features(df, fine_grained_HP_types=False):
                 ("heater" in heating),  # not specified heater
             ]
 
-            # Warm air
-            # --------------------------
-
-            if "warm air" in heating:
-                system_type = "warm air"
-                source_type = "electric"
-                has_hp = False
-
             # Different heat pump types
             # --------------------------
 
-            elif "ground source heat pump" in heating:
+            if ("ground source heat pump" in heating) or (
+                "ground sourceheat pump" in heating
```

Comment:
added this as I found a few instances of "ground sourceheat pump" and "air sourceheat pump" when looking at the `MAINHEAT_DESCRIPTION` variable.

Reply:
For completeness' sake, maybe also add `groundsource heat pump`?
In the future we might want to strip all the white spaces and use this for comparison.